tests/test_apps/test_pictures/test_fetch_pictures.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from server.apps.identity.container import container
+from server.apps.pictures.logic.usecases.pictures_fetch import PicturesFetch
+
+
+def assert_correct_pictures(response, limit) -> None:
+    """Assert pictures."""
+    fields = [
+        'id', 'url',
+    ]
+    assert len(response) == limit
+    for picture in response:
+        for field in fields:
+            assert getattr(picture, field, None)
+
```

Comment:
nit: Я бы сторался избегать вложенных циклов.
```suggestion
from server.apps.pictures.intrastructure.services import placeholder

for picture in response:
    assert isinstance(picture, placeholder.PictureResponse)
```

Reply:
о, спасибо! так лучше ;)